rclpy/package.xml

```diff
@@ -42,6 +42,8 @@
   <exec_depend>python3-yaml</exec_depend>
   <exec_depend>rosgraph_msgs</exec_depend>
   <exec_depend>rpyutils</exec_depend>
+  <exec_depend>service_msgs</exec_depend>
+  <exec_depend>type_description_interfaces</exec_depend>
```

Comment:
I believe that this one should be a `depend`, not an `exec_depend`.  In particular, we need this at build-time as well: https://github.com/ros2/rclpy/blob/1ffd2af51ed402b8173ee38c3d1114b025edd19a/rclpy/src/rclpy/type_description_service.cpp#L45